predix/admin/timeseries.py

```diff
@@ -1,6 +1,7 @@
 
 import os
-import urlparse
+
+from future.moves.urllib.parse import urlparse
```

Comment:
Yep, using this one now.  I considered whether to use `six` but seemed to make no difference.